### PR TITLE
Added case insensitive sort

### PR DESCRIPTION
**Pull request by meddyrainzo**
Added the ability to sort the columns in a case insensitve manner. To trigger this, just add 'caseInsensitive': true.

### CODE REVIEW

**Comment by deitch** [comment]
Looks nice. Can you update the `README.md` to show the new flag?

**Comment by meddyrainzo** [comment]
Sure thing

File: README.md [hunk 1]
```diff
@@ -100,7 +100,7 @@ We use the div to control the entire height and width, and the span to get acces
 * `stateful`: true/false. If true, then whenever a column width is resized, it will store it in html5 localStorage, if available. Defaults to false.
 * `contextmenu`: true/false whether or not a context menu for editing the cells should be shown on right-click. Defaults to false.
 * `gridcontextmenu`: function to create context menu items; see context menu of jstree for format. In addition, if `false` or not set, no special context menu. If `true`, creates a default menu to edit the cell entry.
-
+* `caseInsensitive`: true/false whether or not the column sort should be case sensitive or insensitive
```
Comment (deitch) [inline README.md]: There is a formatting error here. Need an extra newline between the `caseInsensitive` bullet point and the new paragraph. 

Also, please indicate the default.

**Comment by deitch** [comment]
Thanks!

**Pull request merged**